doc/conf.py

```diff
@@ -96,6 +103,15 @@
 
 autodoc_docstring_signature = True
 
+
+# Sphinx gallery configuration
+sphinx_gallery_conf = {
+   # path to your examples scripts
+    'examples_dirs' : '../examples',
+    # path where to save gallery generated examples
+    'gallery_dirs'  : 'auto_examples'}
```

Comment:
It's not possible to rename the current examples/mpl_examples to make room for the gallery-based examples? I strongly dislike the `auto_examples` name as well--I've overridden that name for sphinx-gallery in my own projects.